mapsforge-map/src/main/java/org/mapsforge/map/layer/Layer.java

```diff
@@ -23,11 +23,15 @@
 import org.mapsforge.core.model.Point;
 import org.mapsforge.map.model.DisplayModel;
 
+import java.util.logging.Logger;
+
 public abstract class Layer {
     protected DisplayModel displayModel;
     private Redrawer assignedRedrawer;
     private boolean visible = true;
 
+    private static final Logger LOGGER = Logger.getLogger(Layers.class.getName());
```

Comment:
We should use `Layer.class.getName` here.

Reply:
c&p error, fixed